src/popup/pages/SignTransaction.vue

```diff
@@ -51,8 +58,24 @@ export default defineComponent({
           return;
         }
 
+        const replaceCallerTx = async (tx: Encoded.Transaction) => {
+          const unpackedTx = unpackTx(tx);
+          if (unpackedTx.tag === Tag.ContractCallTx || unpackedTx.tag === Tag.ContractCreateTx) {
+            // TxUnpacked isn't exported
+            // typescript is unaware that ContractCallTx and ContractCreateTx do have a callerId
+            (unpackedTx as any & { callerId: Encoded.AccountAddress })
```

Comment:
I suggest to override/set the typing where the constant is created and extend the return type instead of overriding it with `any` because `any & { ... }` is treated as `any`:

```
const unpackedTx = unpackTx(tx)  as ReturnType<typeof unpackTx> & { callerId: Encoded.AccountAddress };
```
(not tested)

Reply:
I'll try, thanks!

Reply:
seems to work :)